engine/crates/graphql-mocks/src/tea_shop.rs

```diff
@@ -0,0 +1,218 @@
+use async_graphql::{Context, EmptyMutation, EmptySubscription, Object, SimpleObject, Union};
+
+use crate::Schema;
+
+/// A schema that only uses String types.
```

Comment:
I see at least one usize so this comment seems slightly inaccurate

Reply:
oh yeah missed that in the copy-paste